Assets/Scripts/Enemy/KitchenEnemy.cs

```diff
@@ -132,6 +139,15 @@ public class KitchenEnemy : MonoBehaviour {
 		SetFacing(turnDirection);
 	}
 
+	public void PlayGruntSound() {
+		gruntSounds.PlayRandom();
+	}
+
+	public void PlaySoupSound() {
+		soupBeamSound.Play();
+
+	}
+
```

Comment:
These can be private.
```suggestion
	private void PlayGruntSound() {
		gruntSounds.PlayRandom();
	}

	private void PlaySoupSound() {
		soupBeamSound.Play();
	}

```

Reply:
the reason for those to be public is because they are used by the animation events, think they have to be public for that right? 

and the reason that I do play the soup sounds both in the function and in the OnAttackStart is because there's a timing difference between the Up attack and the Left Attack animation.

Reply:
No, animation events do not require the receiver to be public. All my animation events in that script are private.

Yea, I figured it was a timing thing. I was thinking that the sound should play when the "beam" is spawned for both the up and the side attack. But maybe that didn't sound right.